The pipeline build is failing at the lint check:
```
pkg/cmd/ssh_aws.go:100:2: ineffectual assignment to `command` (ineffassign)

	command := make([]string, 0)

	^

pkg/cmd/ssh_azure.go:84:2: ineffectual assignment to `command` (ineffassign)

	command := make([]string, 0)

	^

pkg/cmd/ssh_gcp.go:89:2: ineffectual assignment to `command` (ineffassign)

	command := make([]string, 0)
```

--- a/pkg/cmd/ssh_aws.go
+++ b/pkg/cmd/ssh_aws.go
@@ -97,7 +97,7 @@ func sshToAWSNode(targetReader TargetReader, nodeName, path, user, pathSSKeypair
 		args = append([]string{"-vvv"}, args...)
 	}
 
-	command := make([]string, 0)
+	var command []string
 	if a.FlagProviderID != "" {
 		command = os.Args[4:]
 	} else {

--- a/pkg/cmd/ssh_azure.go
+++ b/pkg/cmd/ssh_azure.go
@@ -81,7 +81,7 @@ func sshToAZNode(targetReader TargetReader, nodeName, path, user, pathSSKeypair 
 		args = append([]string{"-vvv"}, args...)
 	}
 
-	command := make([]string, 0)
+	var command []string
 	if flagProviderID != "" {
 		command = os.Args[4:]
 	} else {

--- a/pkg/cmd/ssh_gcp.go
+++ b/pkg/cmd/ssh_gcp.go
@@ -86,7 +86,7 @@ func sshToGCPNode(targetReader TargetReader, nodeName, path, user, pathSSKeypair
 		args = append([]string{"-vvv"}, args...)
 	}
 
-	command := make([]string, 0)
+	var command []string
 	if flagProviderID != "" {
 		command = os.Args[4:]
 	} else {

--- a/pkg/cmd/ssh_openstack.go
+++ b/pkg/cmd/ssh_openstack.go
@@ -93,7 +93,7 @@ func sshToOpenstackNode(nodeName, path, user, pathSSKeypair string, sshPublicKey
 		args = append([]string{"-vvv"}, args...)
 	}
 
-	command := make([]string, 0)
+	var command []string
 	if flagProviderID != "" {
 		command = os.Args[4:]
 	} else {

